packages/jellyfish-transaction-builder/src/txn/txn_builder_loans.ts

```diff
@@ -197,12 +201,12 @@ export class TxnBuilderLoans extends P2WPKHTxnBuilder {
   }
 
   /**
-   * PaybackLoan to vault transaction.
-   *
-   * @param {PaybackLoan} paybackLoan txn to create
-   * @param {Script} changeScript to send unspent to after deducting the (converted + fees)
-   * @returns {Promise<TransactionSegWit>}
-   */
+     * PaybackLoan to vault transaction.
```

Comment:
This appears as expected on my machine, not sure why it shows an extra space here. Don't think its the problem with spacing, maybe something wrong with the diff?

Reply:
<img width="286" alt="Screenshot 2022-05-04 at 12 01 30 AM" src="https://user-images.githubusercontent.com/7101806/166490784-58e55868-dd0c-4356-891f-873bda917346.png">

I use webstorm, and this is how it appeared to me.

Reply:
Weird, maybe vscode has problems then. I'll try to see if I can fix this.